### PR TITLE
585 add vertical profile

### LINKED ISSUE #585: Add vertical profile
Previously handled in #355 
Depends on #587

### PR DESCRIPTION
**Pull request by hvangeffen**
Closes #585

### CODE REVIEW

File: src/components/charts/ElevationChart.vue [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+      return {
+        id: id,
+        name: name || '',
+        disabled: false,
+        legendSvg: s.serializeToString(legendSvg),
+      }
+    })
+  }
+
+  function createChip() {
```
Comment (hvangeffen) [inline src/components/charts/ElevationChart.vue:275]: Remove chips

File: src/components/charts/ElevationChart.vue [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+    legendSvg.setAttribute('width', '20')
+    legendSvg.setAttribute('height', '20')
+    legendSvg.setAttribute('viewBox', '0 0 20 20')
+    const svgGroup = document.createElement('g')
+    svgGroup.setAttribute('transform', 'translate(0 10)')
+    return { svgGroup, legendSvg }
+  }
+}
+
+const toggleLine = (id: string) => {
```
Comment (hvangeffen) [inline src/components/charts/ElevationChart.vue:286]: Remove

File: src/components/spatialdisplay/SpatialTimeSeriesDisplay.vue [hunk 3]
```diff
@@ -30,7 +30,12 @@
             <v-icon>mdi-close</v-icon>
           </v-btn>
         </template>
-        <TimeSeriesComponent :config="displayConfig" :displayType="displayType">
+        <TimeSeriesComponent
+          :config="displayConfig"
```
Comment (hvangeffen) [inline src/components/spatialdisplay/SpatialTimeSeriesDisplay.vue:34]: I think the displayConfig is an Array already

File: src/components/charts/ElevationChart.vue [hunk 4]
```diff
@@ -0,0 +1,384 @@
...
+  }
+  if (props.config.xAxis) {
+    axis.setOptions({
+      x: [props.config.xAxis[0], props.config.xAxis[1]],
+    })
+  }
+
+  axis.redraw({
+    x: {
+      autoScale: true,
```
Comment (hvangeffen) [inline src/components/charts/ElevationChart.vue:239]: Should be something like nice in case of defaultDomain

**Comment by wkramer** [comment]
Contains a number of changes that should be reverted. Do not merge as is.

**Comment by hvangeffen** [comment]
Vertical mouseOver is being added in https://github.com/Deltares/fews-web-oc-charts/pull/79

**Comment by wkramer** [comment]
Still has issues with switching locations and between timeseries chart and vertical profile.

**Pull request merged**